### DISCUSSION TITLE
Many-to-many label matching: labels differ on the two sides and the calculation fails [need help]

### DISCUSSION THREAD

**Discussion by harvey945** [ccfos/nightingale#1192] [Q&A]
我想把进程的内存占用百分比计算出来，查询语句如下：
![5d8db8dff9d3c7e2560746e63c6225c](https://user-images.githubusercontent.com/93895686/195525952-e7ae40ae-3984-425c-ad1b-b02590f2efed.jpg)

请大神不吝赐教。
先行拜谢。

**Comment by harvey945** [comment]
在grafana中可以使用变量保持统一。在PromQL中如何保持统一呢？
![c3d0b7aa5e66bb53db7e3b935e501f2](https://user-images.githubusercontent.com/93895686/195526592-6905b9d5-4612-45cc-b27d-2b715250c7a9.jpg)

**Comment by UlricQin** [comment]
那个报错是Prometheus返回的，不支持多对多的表达式。你是想求取内存利用率之类的这种指标吧，这个可以直接使用categraf或者telegraf来监控windows，自带这种指标，不用二次计算

**Reply by harvey945** [reply]
谢谢。
